build-packages/xmldom/readme.md

```diff
@@ -0,0 +1,237 @@
+# XMLDOM
```

Comment:
Ok, so about this module, let's do the same we did for the "Craco" - please remove all code not related to us - .github, all test files, keep just the source, package.json and README. Replace README with text explaining why this package exists, i.e.:

Reply:
```md
This package exists, because it implements un-merged features for https://github.com/gsoft-inc/craco/pull/175 PR. This package will be abandoned and replaced with original Craco package as soon as the PR is merged.
```

Reply:
Done